src/frontend/overlay/src/components/organisms/widgets/FullScreenClock.tsx

```diff
@@ -1,13 +1,22 @@
 import React from "react";
 import ContestClock from "../../molecules/Clock";
-import styled from "styled-components";
+import styled, { css } from "styled-components";
 import c from "../../../config";
 
 const ClockWrapper = styled.div`
   display: flex;
   justify-content: center;
 
-  padding-top: 240px;
+  ${c.FULL_SCREEN_CLOCK_PADDING_TOP == "center"
+        ? css`
+                align-items: center;
+                width: 100%;
+                height: 100%;
+            `
+        : css`
+                padding-top: ${c.FULL_SCREEN_CLOCK_PADDING_TOP};
+            `
+}
```

Comment:
I don't really like that ```FULL_SCREEN_CLOCK_PADDING_TOP``` can be the "center" and the number of pixles.

Perhaps you should enter another constant in the config ```IS_FULL_SCREEN_CLOCK_CENTERED```. Then ```FULL_SCREEN_CLOCK_PADDING_TOP``` can be a number, not a string.

Reply:
Okay, but then which parameter should take precedence?
For example, we have FULL_SCREEN_CLOCK_PADDING_TOP and IS_FULL_SCREEN_CLOCK_CENTERED in a config. What should the program do? I see three options here:
1. Just add padding to the clock
2. Just center the clock
3. Center and add padding to the clock

Reply:
If our value `IS_FULL_SCREEN_CLOCK_CENTERED` is true, we will center the clock. Just do something like this:
```
${c.IS_FULL_SCREEN_CLOCK_CENTERED
     ? css`
          // your code
        `
      : css`
           // your code
`
}
```